### PR TITLE
Cleans up CSS post flair-site golive

### PR DESCRIPTION
**Pull request by Badspler**
Cuts flair CSS to just compress the tracker url name and reveal on hover.

Additionally means free form text on old can't be displayed as there is nothing to hover over without an emoji. Thats a good thing if someone picks the "go set your flair" flair. 

Not sure if free-form flairs are needed just yet (like verified users)?

### CODE REVIEW

**Review by durinthal — APPROVED** [review]

File: src/_userflair.scss
Comment (eritbh) [inline src/_userflair.scss:7]: Minor nitpick: This can be simplified since we're not trying to set a different value for certain sides anymore

```suggestion
	padding: 0;
```

**Review by eritbh — APPROVED** [review]

File: src/_userflair.scss
Comment (eritbh) [inline src/_userflair.scss:55]: After checking the site, I realized there's no space between the emojis and the flair itself, which feels kinda cluttered, and removing this line will also remove the space at the end of the flair, which is more consistent but doesn't really help. With no margin the flairs look like this for me:

image

And here's what they look like with the 4px spacing on the sides, which I find significantly easier to read and not significantly bulkier:

image


A better solution would be to leave this line in, and add another block that adds back the margin between icon and text that, like so:

```scss
.flair:hover > :last-child {
  margin-left: 5px;
}
```

Since we have tight control over the formatting of flairs with the new system,`> :last-child` to pick out the plain text section of the flair should work; you can also use `> :not([class])` to select *all* text nodes, which shouldn't make a big difference here but may be useful to know in the future.

File: src/_userflair.scss
Comment (eritbh) [inline src/_userflair.scss:16]: This line should be removed to avoid the flair icons disrespecting the border radius of the flair itself.

**Review by eritbh — CHANGES_REQUESTED** [review]
On second review, after actually pulling out my laptop and testing this out on the sub, found a couple minor visual discrepancies with the current code. I've left them as suggestions here, but I can also send my own PR after the fact to address these if that's preferred (duri should be able to dismiss this review if it ends up blocking merging in that case).

File: src/_userflair.scss
Comment (Badspler) [reply src/_userflair.scss:55]: I agree with the above. 

I am also seeing a separate issue on staging where after manually setting from the reddit mod pages, no flair is displayed whatsoever.

**Comment by eritbh** [comment]
definitely didn't forget this PR existed for 3 months, nooooo, that doesn't sound like something I would do

**Review by eritbh — APPROVED** [review]

**Pull request merged**
